hyperspy/_components/eels_cl_edge.py

```diff
@@ -69,27 +69,45 @@ class EELSCLEdge(Component):
         favourable cases is proportional to the number of atoms of
         the element. It is a component.Parameter instance.
         It is fixed by default.
-    fine_structure_coeff : Parameter
-        The coefficients of the spline that fits the fine structure.
-        Fix this parameter to fix the fine structure. It is a
-        component.Parameter instance.
     effective_angle : Parameter
         The effective collection semi-angle. It is automatically
         calculated by set_microscope_parameters. It is a
```

Comment:
```suggestion
        calculated by ``set_microscope_parameters``. It is a
```

In the following, a number of parameter references are just italics, should they also be code or even explizit sphinx-references?